### ISSUE TITLE
Deprecation warning thrown by webpack 2.

### ISSUE THREAD

**Issue by rajatbarman** [epegzz/sass-vars-loader#16]
DeprecationWarning: loaderUtils.parseQuery() received a non-string value which can be problematic, see https://github.com/webpack/loader-utils/issues/56
parseQuery() will be replaced with getOptions() in the next major version of loader-utils.

Trace - 
```
at Object.parseQuery (/home/ubuntu/toppr-web-v4/client/node_modules/loader-utils/index.js:78:3)
    at Object.exports.getLoaderConfig (/home/ubuntu/toppr-web-v4/client/node_modules/loader-utils/index.js:121:22)
    at Object.loader (/home/ubuntu/toppr-web-v4/client/node_modules/@epegzz/sass-vars-loader/index.js:12:30)
    at LOADER_EXECUTION (/home/ubuntu/toppr-web-v4/client/node_modules/loader-runner/lib/LoaderRunner.js:119:14)
    at runSyncOrAsync (/home/ubuntu/toppr-web-v4/client/node_modules/loader-runner/lib/LoaderRunner.js:120:4)
    at iterateNormalLoaders (/home/ubuntu/toppr-web-v4/client/node_modules/loader-runner/lib/LoaderRunner.js:229:2)
    at Array.<anonymous> (/home/ubuntu/toppr-web-v4/client/node_modules/loader-runner/lib/LoaderRunner.js:202:4)
    at Storage.finished (/home/ubuntu/toppr-web-v4/client/node_modules/enhanced-resolve/lib/CachedInputFileSystem.js:40:15)
    at /home/ubuntu/toppr-web-v4/client/node_modules/enhanced-resolve/lib/CachedInputFileSystem.js:77:9
    at /home/ubuntu/toppr-web-v4/client/node_modules/graceful-fs/graceful-fs.js:78:16
    at FSReqWrap.readFileAfterClose [as oncomplete] (fs.js:511:3)
```

**Comment by epegzz** [comment]
Do you still encounter this warning? If so, do you mind pasting the sass-vars-loader config section from your webpack config file?

**Comment by epegzz** [comment]
Can you check in your package.json which version of `sass-vars-loader` you have installed?

**Comment by epegzz** [comment]
I released version `3.0.0` yesterday which is a complete rewrite including updated dependencies. Try upgrading to that version. If you still get this error, feel free to re-open this issue :)

**Comment by rajatbarman** [comment]
Thanks, that fixed it.
